2_adventures/003/wavrin/15/README.md

```diff
@@ -0,0 +1,10 @@
+Today I thought a lot about managing ruby versions. I'd recently upgraded computers and wanted to change how I managed ruby versions.
+
+On previous computer I used rvm + gemsets. But that seemed to get a bit hairy.
+
+On new computer, I'm using just new version of ruby and letting Bundler handle gem versions.
```

Comment:
If you use homebrew, ```ruby-install``` is a great utility and works quite well with ```chruby``` to provide multiple rubies on your machine (installed in ~/.rubies/ so they're easy to nuke).